tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -2225,6 +2225,13 @@ def depersonalize(value: str, key: str):
     dir_keys = [
         'output_dir', 'extensions', 'saved_model_dir', 'tensorboard_logdir', 'caffe_parser_path'
     ]
+
+    if isinstance(value, list):
```

Comment:
If `value` can be `list` you need to change function signature, currently value is stated as `str`.